hub/config.go

```diff
@@ -32,6 +32,9 @@ func SetConfigDefaults(v *viper.Viper) {
 	v.SetDefault("metrics", false)
 	v.SetDefault("metrics_login", "mercure")
 	v.SetDefault("metrics_password", "")
+
+	v.SetDefault("telemetry.enabled", false)
+	v.SetDefault("telemetry.addr", "127.0.0.1:3002")
```

Comment:
Shouldn't we register a dedicated default port for Mercure here: https://github.com/prometheus/prometheus/wiki/Default-port-allocations ?

Reply:
Shouldn't we change the port to one we'll add to this list @sagikazarmark?

Reply:
9755 for instance

Reply:
It doesn't matter on Kubernetes, you can change it from config everywhere else. IMHO it doesn't matter much what the default port is, it's up to you.